docs/rfcs/0004-suggested-excluded-apps-tech-spec.md

```diff
@@ -0,0 +1,141 @@
+
+Other options considered included a fixed list of apps to suggest, which could only be updated when the app updates.
+
+We considered hosting the list in Guardian (which is a private repo, so the list would need to be hosted elsewhere to allow public suggestions) and in the client repo outside of addons (requiring a new app release to update the list).
+
+### Other options considered for suggested app files
+
+The goals for the file format:
+-   Easy to read by humans
+-   Easy for humans to add to, no matter their level of technical proficiency
+-   Easy for us to parse in the app
+
+Initially, there were two additional goals (no longer considerations):
+-   To have one file for all platforms, so that we'd remember to add a specific app to all platforms at once. With a conversation around potentially wanting different suggested app categories on mobile, this seems unimportant.
+-   Allowing comments in the file. While we still could structure a text file to allow comments, this became less important when the plan became "one file per platform, one app per line".
+
+Other options considered included CSV, YAML, and JSON. While YAML could be useful (as it's easy for less-technical people to read), we don't have a good YAML parser in the app yet. We could add a library, of course. However, we'd need to write our own document format test, because the YAML file (or JSON or CSV) would need to be structured in a specific way.
+
+Ultimately, the multiple text files seemed simpler (and thus easier to build) - there aren't the same parsing concerns and it's easy for contributors.
+
+Metrics
+-------
+
+We will create several metrics as part of this work; they are described in the appropriate spots in the "Implementation" section.
+
+We'd expect the `apps_excluded` metric to increase after this is implemented on platforms which have app exclusions. Both the number of people with more than 0 apps excluded and the average count of apps excluded should increase.
+
+If Nimbus is available by the time this feature is complete, we will use it to roll out the test.
+
+Security / Privacy
+------------------
+
+Since all the work is being done on device, the only security concerns are ensuring we don't log any private info.
+
+The excluded apps list will likely live in the Guardian repo. There are ways to weaponize the suggested apps list to weaken privacy of key apps, but for this attack vector to be used a Mozilla staffer would need to approve the PR with the bad-faith addition to the suggested app list.
```

Comment:
Should we update this paragraph? On line 47 the suggestion is to keep the list in the client repo.

Reply:
Yes, thanks! I missed a spot. Will be updated with the next push.